.github/workflows/build_on_release.yml

```diff
@@ -41,14 +41,27 @@ jobs:
         mkdir -p commands
         go run ./cmd gendoc
 
+    - name: convert MD to HTML
```

Comment:
How do we provide styling? Can you attach tailwind CSS to the style? The CSS is available here https://cdnjs.cloudflare.com/ajax/libs/tailwindcss/2.2.19/tailwind.min.css